.github/workflows/cve_scan_runner.yml

```
@@ -0,0 +1,36 @@
+name: cve-scan-runner
+
+on:
+  workflow_dispatch:
+
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  # To see the script output in real time
+  PYTHONUNBUFFERED: 1
+
+jobs:
+  scan-and-open-issues:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Clone the osquery repository
+        uses: actions/checkout@v3
+
+      - name: Install python pre-requisites
+        run: |
+          pip3 install nvdlib==0.6.0 pygit2
+
+      - name: Scan CVEs and open issues
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        # NIST_API_KEY: ${{ secrets.NIST_API_KEY }}
```

This is https://nvd.nist.gov/developers correct?

I have registered osquery for an API key. Added it to 1password, and added it to github secrets as `NVD_API_KEY`

```suggestion
        NIST_API_KEY: ${{ secrets.NVD_API_KEY}}
```